bin/rialto-parachain/runtime/src/lib.rs

```diff
@@ -588,7 +588,7 @@ construct_runtime!(
 		System: frame_system::{Pallet, Call, Storage, Config, Event<T>},
 		Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent},
 		Sudo: pallet_sudo::{Pallet, Call, Storage, Config<T>, Event<T>},
-		RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Storage},
+		RandomnessCollectiveFlip: pallet_insecure_randomness_collective_flip::{Pallet, Storage},
```

Comment:
@svyatonik 
do you know what is this about and if we need this? I have no idea :)
I just saw that they removed it from statemine/t:  https://github.com/paritytech/cumulus/blob/master/parachains/runtimes/assets/statemint/src/lib.rs#L607

and leave it just for constracts-rococo: https://github.com/paritytech/cumulus/blob/master/parachains/runtimes/contracts/contracts-rococo/src/lib.rs#L335